src/services/serverApi.ts

```diff
@@ -83,34 +83,35 @@ export const {
 } = serverApi;
 
 export const useGetMembers = () => {
-  const { data, isLoading, isFetching, error } = serverApi.useGetMembersQuery()
-  const membersWithRole = data?.members?.filter(
-    (member: MemberType) =>
-      member?.isMember === true && member?.first_name && !member.roles.archived
+  const { data, isLoading, isFetching, error } = serverApi.useGetAllUsersQuery();
+
+  const usersWithMemberRole = data?.users?.filter(
+    (member: UserType) =>
+      member?.roles.member === true && member?.first_name && !member.roles.archived
   );
-  const sortedUsers = membersWithRole?.sort((a,b) => a.first_name > b.first_name ? 1 : -1) 
 
+  const sortedMembers = usersWithMemberRole?.sort((a, b) => a.first_name > b.first_name ? 1 : -1)
```

Comment:
- What is the purpose of a sort here?

Reply:
To show the Members in an Alphabetical Order w.r.t their first name.

Reply:
- please add a comment stating the same